blueflood-elasticsearch/src/main/java/com/rackspacecloud/blueflood/io/ElasticsearchRestHelper.java

```diff
@@ -0,0 +1,266 @@
+package com.rackspacecloud.blueflood.io;
```

Comment:
this file, blueflood-elasticsearch/src/main/java/com/rackspacecloud/blueflood/io/ElasticsearchRestHelper.java
, has a number of intellij warnings.  please take a look at those and see if they need handling.